doc/recording.md

```diff
@@ -0,0 +1,369 @@
+# Recording
+As stated on the [RAAR Import Recordings section](import.md#recordings), RAAR
+is not responsible for the actual recording of the audio files. It simply
+assumes that recordings are placed within directories defined by
+`IMPORT_DIRECTORIES` waiting for the [RAAR Importer](import.md) to pick them
+up.
+
+But don't panic :-) we've got you covered in case you haven't your own
+recording solution already in place.
```

Comment:
I'm not quite sure but something seems wrong about this sentence. It's probably "haven't" that triggers me...

"we've got you covered in you don't have preexisting recording solution"